test/passing/source.ml.ref

```diff
@@ -3452,7 +3451,7 @@ module SSMap = struct
 end
 
 let ssmap =
-  ( module SSMap : MapT
+  (module SSMap : MapT
```

Comment:
This is unavoidable as the opening and closing parentheses are not in the same box (but most of the case the output is consistent with `indicate-multiline-delimiters`).

Reply:
This is still a bug. There is a space before the closing parenthesis.

Reply:
I agree, but that's the kind of things that can be improved later I think and it shouldn't delay this PR that improves a lot of things.
We need to rethink the `indicate-multiline-delimiters` because in most of the cases it is not feasible without relying on the concrete input (which is bad).

Reply:
I agree.